src/cli.ts

```diff
@@ -105,15 +109,30 @@ async function run(verb: string, files: string[]): Promise<boolean> {
     return init(options);
   }
 
-  const lint: VerbFilesFunction = require('./lint').lint;
-  const format: VerbFilesFunction = require('./format').format;
+  const flags = Object.assign({}, files);
+  if (flags.length === 0) {
+    flags.push('**/*.ts', '**/*.js');
+  }
+
   switch (verb) {
-    case 'check':
-      const passLint = await lint(options, files);
-      const passFormat = await format(options, files);
-      return passLint && passFormat;
+    case 'check': {
+      try {
+        await execa('npx', ['eslint', ...flags], { stdio: 'inherit' });
```

Comment:
@bcoe @google/google-node-team so I'm interested in feedback on this one, because I'm a bit torn.  I'm experimenting with getting rid of the wrapping invocation of `prettier` and `tslint`, and instead just shelling out to `eslint`.  I'm doing this for a few reasons:
- With `tslint` and `prettier`, we had to run the format and lint command separately.  With `eslint-prettier`, we can combine them into a single call now.
- The fewer moving parts in this library the better.  Since `eslint` pretty much does what we want, just shelling out seems like a lot less can go wrong.  

That having been said - I recognize there are probably ways this could go south.  Specifically testing it will be much harder 😆   What am I missing here?  What do others think?